packages/react-examples/src/react/ComboBox/ComboBox.Inline.Example.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { ComboBox, IComboBoxOption, IComboBoxStyles } from '@fluentui/react';
+
+const options: IComboBoxOption[] = [
+  { key: 'A', text: 'Option A' },
+  { key: 'B', text: 'Option B' },
+  { key: 'C', text: 'Option C' },
+  { key: 'D', text: 'Option D' },
+];
+// Optional styling to make the example look nicer
+const comboBoxStyles: Partial<IComboBoxStyles> = { root: { maxWidth: 300 } };
+
+export const ComboBoxInlineExample: React.FunctionComponent = () => {
+  return (
+    <div>
+      <ComboBox
+        defaultSelectedKey="C"
+        label="Combobox with inline options list"
+        options={options}
+        styles={comboBoxStyles}
+        calloutProps={{ doNotLayer: true }}
+      />
+      <div
+        // since this example is an inline picker, it needs some forced space below
+        // so when embedded in an iframe in the website, the dropdown shows up
```

Comment:
It's not technically an iframe, just a container element. Might also be worth mentioning if/when people would need to do something similar in their own app.

Reply:
Ah, yup! I even knew and double-checked that it wasn't an iframe, and somehow managed to write this anyway 🤦‍♀️. Fixed now!